src/eventloop.c

```diff
@@ -215,24 +216,24 @@ guint input_add(int fd, PurpleInputCondition cond,
 */
 gboolean input_remove (guint int_handle) {
     gpointer handle = GUINT_TO_POINTER(int_handle);
+    g_warning("input_remove: removing %p (%u)", handle, int_handle);
     g_return_val_if_fail(handle != NULL, false);
     s_evLoopInputEvent *inputEvent = handle;
     s_evLoopInput *input = inputEvent->parent;
+    // Why is this here? XXX
     if (g_list_find(input->events, inputEvent) == NULL) {
```

Comment:
I think this is used to check if the input actually exists in our hash, but granted it's probably pointless here.